netkan/pyproject.toml

```
@@ -0,0 +1,112 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "netkan_indexer"
+version = "1.0"
+dependencies = [
+    "boto3",
+    "click",
+    "gitpython",
+    "pynamodb",
+    # 2019-11-01 capping to 2.8.0 - https://github.com/boto/botocore/commit/e87e7a745fd972815b235a9ee685232745aa94f9
+    "python-dateutil>=2.1,<2.8.1",
+    "requests",
+    "flask",
+    "jinja2",
+    "internetarchive!=3.0.1",
+    "gunicorn>=19.9,!=20.0.0",
+    "discord.py>=1.6.0,<=1.7.3",
+    "PyGithub",
+    "ruamel.yaml",
+]
+requires-python = ">=3.8"
+authors = [{ name = "Leon Wright", email = "techman83@gmail.com" }]
+description = "NetKAN Indexer"
```

```suggestion
description = "NetKAN Infra"
```

Similar comment as above, this isn't just the indexer.

---

Same deal, straight copy paste from existing. Will update.